client/src/pages/ViewEvent/VolunteersTable.tsx

```diff
@@ -372,10 +375,11 @@ export const VolunteersTable: React.FC<{
           "Participant Type",
           "Confirmed",
           "Can't Come",
+          "Past Events",
           "Special Group",
           "Delivery Count",
           "Contact",
-          "Edit",
+          "Edit"  
```

Comment:
I think convention is to keep trailing commas in last element. makes it a little easier over time to add/remove rows and not worry about removing the last comma of the last element.